### ISSUE TITLE
[SCUS-97128] Drakan - The Ancients' Gate - Long load times

### ISSUE THREAD

**Issue by trostboot** [PCSX2/pcsx2#3659]
***PCSX2 version:***
PCSX2 1.7.0-20200830215020, also tested with 1.0.0

***Plugins used:***
defaults, GSdx SW

***Description of the issue:***
Load times, especially initial load from memory card, are very long, and what should be a fairly smooth fade to black after initiating the load is fairly stuttery (emulated FPS stays at 60, however).
In fact, load times are up to twice as long as on real hardware. I've timed it (from pressing the button to load the save to the loading bar disappearing), loading the same save every time, here are the results:

Real fat PS2, internal IDE HDD: 13.09s
PCSX2, no hacks: 25.12s
PCSX2, fast DVD: 17.13s
PCSX2, fast DVD, EE 300%: 15.26s
PCSX2 1.0.0, no hacks: 25.40s

The optical disc drive in my PS2 is pretty sketchy, so no use timing that.
The 1.0.0 result is well within margin of error, but the results are clear. 
And just to point it out, whether the image is on spinning rust or an NVMe SSD makes no difference, host I/O clearly isn't the bottleneck here.

***How to reproduce the issue:***
Load the game! 
[Here is the save I used for this.](https://github.com/PCSX2/pcsx2/files/5155827/Drakan.zip)
For funsies, have PCSX2 and your PS2 running next to each other and press load at the same time. ;)

***PC specifications:***
Ryzen 3900X, GTX1080, Win10 2004

**Comment by MrCK1** [comment]
@trostboot can you double check on the latest dev build to see if anything has changed regarding this?

**Comment by trostboot** [comment]
Same as before, no changes.

**Comment by MrCK1** [comment]
@trostboot can you check this again on the latest dev build? There were a number of CDVD changes made recently that may have affected this issue.

**Comment by trostboot** [comment]
> so using an internal HDD on PCSX2 is faster than the real hardware with a HDD? 

It is, but that's also comparing a PCIe4 NVMe SSD vs. an old IDE drive, so, apples and oranges, as you said. ;)

If someone has the game and a non-croaky drive, I'd love to see the numbers.

**Comment by refractionpcsx2** [comment]
I don't care what OPL says, it isn't accurate :P You can't play Shadowman 2 on OPL without a patch because it reads discs too fast.

**Comment by refractionpcsx2** [comment]
okay, so your issue isn't that the CDVD timings aren't accurate? you're just asking for a "super fast loading method hack"? because right now you're basically telling me "I don't care what the proper speed is, my games don't mind loading quickly so I'm happy with that".

I'm sorry but I'm not in the business of just adding hacks willy nilly and throwing compatibility out the window. Just because most of your games are fine, there will be many which aren't, and our goal is accuracy to the console.

If you need to speed up game loading, I suggest you press F4 to take the frame limiter off, then your game will load in mere seconds.

**Comment by trostboot** [comment]
Never mind, you're clearly not interested in reading what I'm writing. I already went over that.

**Comment by refractionpcsx2** [comment]
I did read what you wrote, you said your library doesn't care about being loaded quickly, your initial post was complaining our disc loading speed wasn't as fast as your PS2 hard drive, nothing you have said has told me you're actually interested in accurate disc reading speed.

**Comment by trostboot** [comment]
No, the issue (in my mind when writing the issue) wasn't the stuttering, it _was_ the load times. The way Drakan transitions between screens during loading seems to be hard-tied to the overall progress of the load bar. Something like advance by 1 frame for every 0.5% progress (numbers completely made up), so it will be smoother when loading is faster.
That's what got my attention, because that wasn't happening on my PS2, which lead me to quantify the differences in load time, thinking that was the problem.

_But as you've rightly pointed out and we have now established, this is pretty much working as intended._ The original issue is therefore resolved (or rather, wasn't one to begin with), and is why I offered to amend the issue to reflect the turn the discussion had taken if there was interest in talking about that further.

Anything else I'm frankly no longer interested in addressing, you made it clear you're not interested in additions not focusing on accuracy, and we're just going in circles anyway.
